stream-compositions/cursors/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>stream-compositions</artifactId>
+    <groupId>com.backbase.stream</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.backbase.stream.compositions</groupId>
+  <artifactId>cursors</artifactId>
+
+  <packaging>pom</packaging>
+
+  <name>Stream :: Compositions :: Cursors</name>
+  <modules>
+    <module>transaction-cursor</module>
+  </modules>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>clean-database</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>sql-maven-plugin</artifactId>
+            <inherited>false</inherited>
+            <version>1.5</version>
+            <configuration>
+              <printResultSet>true</printResultSet>
+            </configuration>
+            <executions>
+              <execution>
+                <id>drop-db</id>
+                <phase>clean</phase>
+                <goals>
+                  <goal>execute</goal>
+                </goals>
+                <configuration>
+                  <driver>com.mysql.jdbc.Driver</driver>
+                  <url>jdbc:mysql://localhost:3306?useSSL=false</url>
+                  <username>root</username>
+                  <password>root</password>
+                  <forceMojoExecution>true</forceMojoExecution>
+                  <sqlCommand>
+                    DROP
+                    DATABASE IF EXISTS transaction_cursor;
+                  </sqlCommand>
+                </configuration>
+              </execution>
+              <execution>
```

Comment:
Remember that my using liquibase it would be useful in both a productive environment and a local development environment. By using this plugin here you are enabling automatic database migration execution only in local environments dependent on a maven extra step.

Reply:
There is a separate PR open for Liquibase support - https://github.com/Backbase/stream-services/pull/147